routes/api/sites.js

```diff
@@ -16,7 +16,9 @@ router.get('/', async (req, res) => {
 
 // Get a site by id
 router.get('/:id', async (req, res) => {
-  const site = await models.Site.findByPk(req.params.id);
+  const site = await models.Site.findByPk(req.params.id, {
+    include: [models.NutritionPartner],
```

Comment:
This include option in the findByPk query tells sequelize to also look up all associated NutritionParner records through the join model. All the NutritionPartner objects are in an array called NutritionPartners... the custom toJSON function just picks out and returns NutritionPartnerIds so make the data a bit more compact.

Reply:
I changed my mind and removed the custom toJSON() function. The default toJSON() will just return all the NutritionPartners as full objects. I think this will make it easier for you on the client side, since you will have the full object to render data from, rather than having to look up from ids.